src/inspector/view/qml/MuseScore/Inspector/InspectorForm.qml

```diff
@@ -68,9 +68,10 @@ Rectangle {
         id: popupController
     }
 
-    StyledFlickable {
+    StyledListView {
         id: flickableArea
         anchors.fill: parent
+        anchors.margins: 12
```

Comment:
This changes the behaviour slightly: previously, the margin was inside the scrollable area, so if you scrolled down you would not see the top margin anymore; now, the margin is around the scrollable area, so when you scroll down, there will be a blank bar at the top, and the scrollbar will also be inset a bit. 

You can solve this by using the margin properties from `Flickable`, rather than `anchors.margins`. See https://doc.qt.io/qt-6.2/qml-qtquick-flickable.html#bottomMargin-prop.

Reply:
I’m confused; these properties seem to be invalid in `StyledListView`.

Reply:
I get an error in the Qt Creator editor for these properties, and the margins don’t seem to be correct in the built application, either:

![Screenshot of inspector](https://github.com/user-attachments/assets/d12705c7-5c85-47bb-9703-ad35d84d05af)